paket.dependencies

```diff
@@ -2,17 +2,16 @@ framework: net45
 source https://www.nuget.org/api/v2
 
 nuget FSharp.Control.AsyncSeq
-nuget MongoDB.Bson ~> 2.0.0
-nuget MongoDB.Driver.Core ~> 2.0.0
+nuget MongoDB.Driver.Core
```

Comment:
FSharp.MongoDB depends on MongoDB.Bson directly, so I don't think we shouldn't have the dependency be inherited transitively through MongoDB.Driver.Core.